joplin/base/migrations/0073_auto_20190604_2124.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.2.2 on 2019-06-04 21:24
```

Comment:
There are a lot of migrations in here, any chance you could do a little bit of git time travel and:
1. Run master locally
2. Pull your branch, delete all migrations
3. Make/run migrations

If that ends up being a mess don't worry about it, just figured it might be a nice way to make these more parseable.